packages/node_modules/@webex/plugin-meetings/src/peer-connection-manager/index.js

```diff
@@ -407,15 +424,16 @@ pc.createAnswer = (params, {meetingId, remoteQualityLevel}) => {
 
   return pc.setRemoteSessionDetails(peerConnection, OFFER, params.offerSdp, meetingId)
     .then(() => peerConnection.createAnswer(params.sdpConstraints))
-    .then((answer) =>
+    .then((answer) =>{
```

Comment:
Not sure how this is passing eslint, but for consistency could you update to:
```suggestion
    .then((answer) => {
```